contracts/W3BC6-Bank.sol

```diff
@@ -33,4 +33,10 @@ contract Bank {
         require(sent, "transacation failed");
         return data;
     }
+
+    function get_bank_balance() external view returns(uint){
+        uint balance;
+        balance = address(this).balance;
+        return balance;
```

Comment:
Great effort! However, there's a vulnerability issue with using address(this).balance when getting a contract balance. You can read more about [that here ](https://github.com/ethereumbook/ethereumbook/blob/develop/09smart-contracts-security.asciidoc#:~:text=This%20sort%20of,with%20unexpected%20balances.)

Something that would work is to return the value of `bank_balance` private state variable. This value is always incremented whenever ether gets into the contract and decremented when there's a transfer of the contract or withdraw.